app/domain/errors.rb

```diff
@@ -307,8 +307,9 @@ module AuthnAzure
           code: "CONJ00053E"
         )
 
-        MissingProviderFieldsInXmsMirid = ::Util::TrackableErrorClass.new(
-          msg:  "Provider fields are missing in xms_mirid {1}",
+        InvalidProviderFieldsInXmsMirid = ::Util::TrackableErrorClass.new(
+          msg:  "Provider fields are in invalid format in xms_mirid {1}." \
```

Comment:
Do we need a `.` here?

Reply:
there is another sentence below so the message that users will get is `Provider fields are in invalid format in xms_mirid {1}. It should have 3 parts`